i18n(pt-BR): Updated `contribute.mdx` (#3755)
withastro/docs: src/content/docs/pt-br/contribute.mdx

Nós recebemos contribuições de qualquer tamanho e contribuidores de qualquer nível de habilidade. Como um projeto open-source, nós acreditamos em dar de volta para nossos contribuidores. Somos felizes em ajudar orientando em PRs, escrita técnica, e em tornar qualquer ideia de funcionalidade em uma realidade.

Gostaria de ficar ainda mais involvido? Veja nosso [Documento de Governança](https://github.com/withastro/.github/blob/main/GOVERNANCE.md) com descrições detalhadas de diferentes cargos, processos de nomeação de mantenedores, processos de revisão de código, e aplicação do Código de Conduta.

## Formas de Contribuir

### Repositórios do projeto

> Existem diversos caminhos para contribuir para o Projeto Astro! Cada repositório Astro possui um README com um link para o arquivo `CONTRIBUTING.md` na raiz do projeto.

Reviewer: Como a gente tá se referindo a diferentes arquivos com o mesmo nome em diversos repositórios, seria mais adequado "um" do que "o"
```suggestion
Existem diversos caminhos para contribuir para o Projeto Astro! Cada repositório Astro possui um README com um link para um arquivo `CONTRIBUTING.md` na raiz do projeto.
```

> Visite o [perfil do Astro no Github](https://github.com/withastro) para encontrar os repositórios de:

Reviewer: Pequeno erro `Github` -> ``GitHub`. Adicionalmente, a estrutura da escrita original não funciona apropriadamente no português, pois fica com um "de" no fim que não concorda com o artigo nos itens, como "o". Eu desviaria um pouco do original no final só para permitir que a estrutura faça sentido no português.
```suggestion
Visite o [perfil do Astro no GitHub](https://github.com/withastro) para encontrar os seguintes repositórios:
```

> - O [código principal do Astro](https://github.com/withastro/astro), incluindo integrações oficiais e modelos iniciais.

Reviewer: Sinto que adicionar "base" aqui ajude a relacionar com a palavra em inglês "codebase" melhor do que só "código"
```suggestion
- A [base de código principal do Astro](https://github.com/withastro/astro), incluindo integrações oficiais e modelos iniciais.
```

- [Documentação do Astro](https://github.com/withastro/docs), um website Astro inteiro! Contribua não somente escrevendo conteúdo, mas também código Astro encaminhando questões de a11y, CSS, UI, e UX. Nós também fazemos nossa documentação disponível em diversos idiomas, então precisamos de ajuda traduzindo o website inteiro.

> - O [compilador Astro](https://github.com/withastro/compiler), escrito em GO, destibuído como WASM.

Reviewer: Acredito que nesse caso faça mais sentido ter o "do" entre as partes.
```suggestion
- O [compilador do Astro](https://github.com/withastro/compiler), escrito em GO, destibuído como WASM.
```

- [Ferramentas da linguagem](https://github.com/withastro/language-tools) do Astro, as ferramentas de editores necessárias para a linguagem Astro (os arquivos `.astro`).

- [Starlight](https://github.com/withastro/starlight), o framework de documentação oficial do Astro.

> - O [Astro Roadmap](https://github.com/withastro/roadmap) onde o futuro do Astro toma forma! Ideias, sugestões e propostas RFC formais para o projeto Astro.

Reviewer: Se não me engano foi utilizado a preposição ao se referir a RFCs em outras páginas, então mudaria isso para ficar padronizado.
```suggestion
- O [Astro Roadmap](https://github.com/withastro/roadmap) onde o futuro do Astro toma forma! Ideias, sugestões e propostas de RFC formais para o projeto Astro.
```

### Tipos de contribuições

Em adição a contribuir com seu próprio código ou conteúdo, você também pode fazer uma grande contribuição se envolvendo ao fazer comentários revisando PRs, adicionando ideias em Issues e Discussions do GitHub e participando em nossas tarefas de manutenção em issues "Fixadas" ("Pinned")! 

Cada PR, especialmente PRs de tradução, precisam de revisores! Revisar PRs e deixar comentários, suggestões ou um "LGTM!" (do Inglês "Looks Good To Me!", ou "Parece Bom Para Mim!" em Português) aprovando é uma ótima forma de começar no Time Documentação e aprender mais sobre Astro.

> Nós também temos uma comunidade bem ativa no [Discord](https://astro.build/chat)! Nós valorizamos as contribuições daqueles que dão boas-vindas a novos membros, respondem perguntas de suporte, e compartilham o que eles tem feito com e para o Astro! Além das contribuições tradicionais no GitHub, o Astro reconhece e suporta membros da comunidade que engajam com a nossa comunidade, compartilham o Astro em postagem de blog, vídeos e palestras em conferencias, e ajudam a manter a saúde da nossa comunidade.

Reviewer: Alguns errinhos de digitação aqui.
```suggestion
Nós também temos uma comunidade bem ativa no [Discord](https://astro.build/chat)! Nós valorizamos as contribuições daqueles que dão boas-vindas a novos membros, respondem perguntas de suporte, e compartilham o que eles tem feito com e para o Astro! Além das contribuições tradicionais no GitHub, o Astro reconhece e suporta membros da comunidade que engajam com a nossa comunidade, compartilham o Astro em postagens de blog, vídeos e palestras em conferências, e ajudam a manter a saúde da nossa comunidade.
```
